docker-compose.yml

```diff
@@ -141,10 +141,10 @@ services:
     environment:
       - PGPASSWORD=dj
     volumes:
-      - ./datajunction-server/init_data:/init_data
+      - ./db-init:/db-init
```

Comment:
Is this just to merge the read-only user creation script and the dj user creation script into a single dir?

Reply:
Yes.